src/main/kotlin/com/github/mitchellolsthoorn/testgenie/evosuite/Runner.kt

```diff
@@ -0,0 +1,133 @@
+package com.github.mitchellolsthoorn.testgenie.evosuite
```

Comment:
Similar to previuos comment: why is the `EvoSuiteRunner` removed entirely and this alternative created from scratch?

Reply:
Same comment as above